src/@gen3/statics/bin/server.ts

```diff
@@ -1,23 +1,36 @@
 import fs = require('fs');
 import * as express from 'express';
+import * as cors from 'cors';
+import * as helmet from 'helmet';
 import s3Proxy = require('s3-proxy');
 
-import {loadConfig, processArgs, LaunchConfig, ProxyConfig} from '../lib/configHelper';
+import { loadConfig, processArgs, LaunchConfig, ProxyConfig } from '../lib/configHelper';
 
 
-function launchServer(configFolder:string) {
+function launchServer(configFolder: string) {
   const app = express();
 
   // enable logging
   app.use(require('morgan')('dev'));
+  app.use(cors());
+  app.use(helmet.default({
+    contentSecurityPolicy: {
+      directives: {
+        "default-src": helmet.contentSecurityPolicy.dangerouslyDisableDefaultSrc,
+        "img-src": null,
+        "script-src": null,
+        "script-src-attr": null
```

Comment:
These are intentional so we can allow existing inline scripts to run, unless we can use `unsafe-inline` as one of the `script-src` directives